src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -623,8 +623,11 @@ public synchronized void applyDML(Sequenced<DMLEvent> sequencedEvent) throws Exc
 
     latestOffset = event.getOffset();
     latestSequenceNum = sequenceNumber;
+
     if (LOG.isTraceEnabled()) {
-      LOG.trace("DML event:{} offset: {} seq num : {}", GSON.toJson(event), latestOffset.get(), latestSequenceNum);
+      LOG.trace("DML event={}, sequenceNumber={}", GSON.toJson(event), latestSequenceNum);
+    } else if (LOG.isDebugEnabled()) {
+      LOG.debug("DML Event={}, sequenceNumber={}", event, latestSequenceNum);
```

Comment:
I would be better to log the event after L#592, as if we will get the event details even if there are any exceptions in processing.